features/simple_sign_in.feature

```diff
@@ -13,7 +13,23 @@ Feature: User simple flows - sign in
   Scenario: User cannot sign in using a disconnected IDP
     Given the user is at Test RP
     And they start a sign in journey
-    Then they cannot sign in with IDP "Stub Idp Demo Three"
+    Then they cannot sign in with IDP "Stub Idp Disconnected"
+
+  Scenario: User can see disconnected IDP hint
+    Given the user is at Test RP
+    And they start a registration journey with IDP "Stub Idp Disconnected"
+    And they submit user details:
```

Comment:
I guess I'm surprised that we have to repeat this boiler-plate everywhere, but fine

Reply:
I'm not thrilled either, but I couldn't find an easy way of putting this into a step.